.gitignore

```diff
@@ -1,2 +1,3 @@
 composer.lock
+.idea
```

Comment:
Please revert this. Ignoring IDE files does not belong to the project rules (otherwise, we would have to support **all** IDEs out there) but to the computer rules.

See https://help.github.com/articles/ignoring-files/ to know how to create a global rule